src/toolbar/toolbar-controller.js

```diff
@@ -59,7 +59,6 @@
 
         normalSizeClick() {
             this.window.restore();
-            this.window.resizeTo(defaultWidth, defaultHeight, 'top-right');
```

Comment:
I think the issue in this case is that for some reason, that when reverting from compact, it becomes unresizable. I'll close this PR for now, since the implementation is likely to be very different.